### PR TITLE
list  server url for proxy for different sdks

### PR DESCRIPTION
**Pull request by yuhao900914**
# Amplitude Developer Docs PR

Adds table for all server URLs for domain proxy by SDK.

## Deadline

When do these changes need to be live on the site?

## Change type

- [X] Doc update.

### CODE REVIEW

File: docs/analytics/domain-proxy.md [hunk 1]
```diff
@@ -100,9 +100,19 @@ This call should return a `200` response code. In the web app, confirm that Ampl
 After the proxy is working correctly, configure the SDK. Amplitude’s SDKs are open source code that you can change. The SDKs already have built-in options to send events to your defined server endpoint.
 
 !!!note
```
Comment (caseyamp) [inline docs/analytics/domain-proxy.md:102]:
```suggestion

```

File: docs/analytics/domain-proxy.md [hunk 2]
```diff
@@ -100,9 +100,19 @@ This call should return a `200` response code. In the web app, confirm that Ampl
...
-    The SDK's don't point to the same API endpoint as the HTTP API V2, but rather a special endpoint for its custom payloads. Send them instead to `https://api2.amplitude.com/`.
-
-- JavaScript(Web): Set the `apiEndpoint` option when initializing the SDK.
-- iOS/Android: Use the `setServerUrl` function to configure the server URL.
+    Different SDKs point to different API endpoint with its custom payload. The following list are the endpoints for different SDKs.
```
Comment (caseyamp) [inline docs/analytics/domain-proxy.md:103]:
```suggestion
The SDKs point to special endpoints for their custom payloads. Find the endpoint for your SDK in this table: 
```

File: docs/analytics/domain-proxy.md [hunk 3]
```diff
@@ -100,9 +100,19 @@ This call should return a `200` response code. In the web app, confirm that Ampl
 After the proxy is working correctly, configure the SDK. Amplitude’s SDKs are open source code that you can change. The SDKs already have built-in options to send events to your defined server endpoint.
```
Comment (caseyamp) [inline docs/analytics/domain-proxy.md:100]: @yuhao900914 sorry for the out of scope suggestion here, but this has always read weird to me. 

Can we rephrase this to something like: 


```suggestion
After the proxy is working correctly, configure your SDK. Amplitude’s SDKs are open source code that you can change and have built-in options to send events to your defined server endpoint.
```

File: docs/analytics/domain-proxy.md [hunk 4]
```diff
@@ -100,9 +100,19 @@ This call should return a `200` response code. In the web app, confirm that Ampl
...
+| [Amplitude-Java](../data/sdks/java/) | `https://api2.amplitude.com/2/httpapi` | Use the `setServerUrl` function to configure the server URL. |
+| [Amplitude-iOS](../data/sdks/ios/) | `https://api2.amplitude.com/` | Use the `setServerUrl` function to configure the server URL. |
+| [Amplitude-TypeScript](../data/sdks/typescript-browser/) | `https://api2.amplitude.com/2/httpapi` | Set the `serverUrl` option when initializing the SDK. |
+| [Amplitude-Kotlin](../data/sdks/android-kotlin/) | `https://api2.amplitude.com/2/httpapi` | Set the `serverUrl` option when initializing the SDK. |
+| [Amplitude-Swift](../data/sdks/ios-swift/) | `https://api2.amplitude.com/2/httpapi` | Set the `serverUrl` option when initializing the SDK. |
+| [Amplitude-Python](../data/sdks/python/) | `https://api2.amplitude.com/2/httpapi` | Set the `server_url` option when initializing the SDK.|
+| [Amplitude-Go](../data/sdks/go/) | `https://api2.amplitude.com/2/httpapi` |  Set the `serverUrl` option when initializing the SDK. |
 
 Now you can send events through your proxy and see them logged in Amplitude.
```
Comment (caseyamp) [inline docs/analytics/domain-proxy.md:118]:
```suggestion
After you configure the SDK, you can send events through your proxy and see them logged in Amplitude.
```

File: docs/analytics/domain-proxy.md [hunk 5]
```diff
@@ -100,9 +100,19 @@ This call should return a `200` response code. In the web app, confirm that Ampl
 After the proxy is working correctly, configure the SDK. Amplitude’s SDKs are open source code that you can change. The SDKs already have built-in options to send events to your defined server endpoint.
 
 !!!note
-    The SDK's don't point to the same API endpoint as the HTTP API V2, but rather a special endpoint for its custom payloads. Send them instead to `https://api2.amplitude.com/`.
-
-- JavaScript(Web): Set the `apiEndpoint` option when initializing the SDK.
-- iOS/Android: Use the `setServerUrl` function to configure the server URL.
+    Different SDKs point to different API endpoint with its custom payload. The following list are the endpoints for different SDKs.
+
+ <div class="big-column">SDK</div> | Endpoint | Set server url |
```
Comment (caseyamp) [inline docs/analytics/domain-proxy.md]:
```suggestion
 |<div class="big-column">SDK</div> | Endpoint | Set server url |
```

fixes broken table

**Review by qingzhuozhen — APPROVED** [review]

**Review by liuyang1520 — APPROVED** [review]

File: docs/analytics/domain-proxy.md [hunk 6]
```diff
@@ -97,12 +97,21 @@ This call should return a `200` response code. In the web app, confirm that Ampl
...
-After the proxy is working correctly, configure the SDK. Amplitude’s SDKs are open source code that you can change. The SDKs already have built-in options to send events to your defined server endpoint.
-
-!!!note
-    The SDK's don't point to the same API endpoint as the HTTP API V2, but rather a special endpoint for its custom payloads. Send them instead to `https://api2.amplitude.com/`.
-
-- JavaScript(Web): Set the `apiEndpoint` option when initializing the SDK.
-- iOS/Android: Use the `setServerUrl` function to configure the server URL.
-
-Now you can send events through your proxy and see them logged in Amplitude.
+After the proxy is working correctly, configure your SDK. Amplitude’s SDKs are open source code that you can change and have built-in options to send events to your defined server endpoint
```
Comment (caseyamp) [inline docs/analytics/domain-proxy.md:100]:
```suggestion
After the proxy is working correctly, configure your SDK. Amplitude’s SDKs are open source code that you can change and have built-in options to send events to your defined server endpoint.
```

**Review by caseyamp — APPROVED** [review]
